docs/pages/model_hub/speaker_recognition.md

```diff
@@ -53,4 +53,9 @@ As an example, let's see how to configure the Pyannote Speaker Diarization deplo
             sample_rate=16000,
         ).model_dump(mode="json"),
     )
-    ```
\ No newline at end of file
+    ```
+
+### Diarized ASR
+
+Speaker Diarization output can be combined with ASR to generate transcription with speaker information. Further details and code snippet are available in [ASR model hub](./asr.md).
+
```

Comment:
Link it to the exact section: `./asr.md/#diarized-asr`.

Reply:
Done